locales/zh-tw.json

```diff
@@ -83,6 +83,7 @@
   "evidence_cannot_take": "階級不夠，無法拿取證物",
   "dumpster": "垃圾桶",
   "crafting_item": "製作 %s",
-  "crafting_bench": "工作台",
+  "crafting_bench": "工作臺",
```

Comment:
```suggestion
  "crafting_bench": "工作台",
```

Reply:
Thank you for your advice, will update now.